### DISCUSSION TITLE
Add an option in the logbook card to hide the text: "triggered by......"

### DISCUSSION THREAD

**Discussion by mstuij** [home-assistant/frontend#15490] [Configuration]
I use the logbook card for showing custom text of my executed automations in Home Assistant/Node-RED. I use the service logbook.log to add a specific text of what the automation has done. For this goal I have created somy dummy entities to which I log.

![image](https://user-images.githubusercontent.com/75884924/219853076-4e7fb5bf-34e4-433c-bff9-117418f2c5b6.png)

The problem is last year it is changed. Now there is always the text: "triggered by service logbook.log".

This pollutes the overview. So it should be very nice if there is an option in the logbook to not show this extra (not relevant) information.

Also options to hide the user ("Supervisor") should be nice and to hide how long ago (only show the time)

**Comment by ildar170975** [comment]
agree

**Comment by kc5uvc** [comment]
I would really like this option as well!

**Comment by melyux** [comment]
Please

**Comment by frapbod** [comment]
I'd also really like this feature. I have bluetooth beacons on my cat's collar and have bluetooth proxies in each room, trying to use an input_text helper and the logbook card to show the last handful of significant chilling locations of each cat. Having the additional 'triggered by' adding 4 lines really clutters it up. 

![XXkJfN4](https://github.com/user-attachments/assets/67742ad3-dff3-4484-a6bd-464823933044)
